docs/assembly.rst

```diff
@@ -153,6 +153,7 @@ If an opcode takes arguments (always from the top of the stack), they are given
 Note that the order of arguments can be seen to be reversed in non-functional style (explained below).
 Opcodes marked with ``-`` do not push an item onto the stack, those marked with ``*`` are
 special and all others push exactly one item onto the stack.
+Opcodes marked with ``H``, ``B`` or ``C`` are present since Homestead, Byzantium or Constantinople, respectively.
```

Comment:
Perhaps the column has better visibility if we do not mark opcodes at all which are available even before homestead. We should also mention that constantinople is still in planning.

Reply:
Oh right, its only `delegatecall` for homestead, everything else is frontier.

Reply:
Updated (also fixed `revert`). Do we want to keep `F` for frontier or remove it?